nf_core/pipeline-template/{{cookiecutter.name_noslash}}/environment.yml

```diff
@@ -6,6 +6,7 @@ channels:
   - bioconda
   - defaults
 dependencies:
+  - conda-forge::python=3.7.3
```

Comment:
Is this really needed in the template?

Reply:
I imagine it was added to force `conda` to use `Python 3` instead of `Python 2`. Maybe having it there as a guide to specify it explicitly when creating your environment isnt a bad idea? Assuming thats why you added it there @ewels ?

See:
https://github.com/nf-core/tools/pull/462